### PR TITLE
Hide duplicate Request Events model aliases

### PR DESCRIPTION
**Pull request by Willxup**
## Summary

- Hide Request Events model aliases when the alias matches the model name.
- Add a regression test for duplicate model alias display.

## Validation

- `npm --prefix ./web run test`
- `npm --prefix ./web run lint`
- `npm --prefix ./web run typecheck`
- `npm --prefix ./web run build`

### CODE REVIEW

File: web/src/components/usage/RequestEventsDetailsCard.tsx [hunk 1]
```diff
@@ -623,8 +623,10 @@ export function RequestEventsDetailsCard({
       const source = String(event.source ?? '').trim() || '-';
       const sourceType = String(event.source_type ?? '').trim();
       const apiKey = String(event.api_key ?? '').trim() || '-';
-      const model = String(event.model ?? '').trim() || '-';
-      const modelAlias = String(event.model_alias ?? '').trim() || '-';
+      const modelValue = String(event.model ?? '').trim();
+      const model = modelValue || '-';
+      const modelAliasValue = String(event.model_alias ?? '').trim();
+      const modelAlias = modelAliasValue && modelAliasValue !== modelValue ? modelAliasValue : '-';
```
[the remark this answers is not in the document]
Comment (Willxup) [reply web/src/components/usage/RequestEventsDetailsCard.tsx:629]: Keeping the exact comparison here. The upstream fallback copies the model value as-is, so case-insensitive matching is unnecessary.
